vars/lvVITester.groovy

```diff
@@ -0,0 +1,5 @@
+def call(Test_path, lvVersion){
```

Comment:
Parameter name should be `testPath`.

Reply:
Change to `def call(testPath, lvVersion){`